pkg/services/sqlstore/migrations/ualert/channel.go

```diff
@@ -26,11 +23,56 @@ type notificationChannel struct {
 }
 
 // channelsPerOrg maps notification channels per organisation
-type channelsPerOrg map[int64]map[interface{}]*notificationChannel
+type channelsPerOrg map[int64][]*notificationChannel
 
 // channelMap maps notification channels per organisation
 type defaultChannelsPerOrg map[int64][]*notificationChannel
 
+// Union type for receiver map that contains both uid -> receiver and ID -> receiver
+type uidOrID interface{}
```

Comment:
There funciton extractChannelIDs that returns interface but it seems that it should return this type. Can you update it please?